workbench-core/example/app/src/configs/staticRouteConfig.ts

```diff
@@ -223,6 +223,14 @@ export const routesMap: RoutesMap = {
       }
     ]
   },
+  [`/authorization/groups/${groupIDRegExpAsString}`]: {
```

Comment:
do all the integ tests pass? We ran into an issue with the group RegExp being the last part of a route before.

Reply:
I had to shuffle them around so they are evaluated in correct order but yes, integration tests are passing.